tools/analyzer_plugin/lib/src/diagnostic/boilerplate_validator.dart

```diff
@@ -69,18 +69,7 @@ class BoilerplateValidatorDiagnostic extends DiagnosticContributor {
   ///
   /// Does not report any errors for the unit, as those are handled when each unit is analyzed
   bool _unitHasDeclarations(CompilationUnit unit, ParsedUnitResult unitResult) {
-    return orbp
-        .getBoilerplateDeclarations(
-            orbp.detectBoilerplateMembers(unit),
-            orbp.ErrorCollector.callback(
-              SourceFile.fromString(unitResult.content, url: unitResult.path),
-              // no-op for these.
-              // It is assumed this method will run for parent files, and the part file will get analyzed in its own context.
-              // Need types on the second args to avoid nullability errors, since they come from a non-null-safe library.
-              onError: (message, [SourceSpan? span]) {}, // ignore: avoid_types_on_closure_parameters
-              onWarning: (message, [SourceSpan? span]) {}, // ignore: avoid_types_on_closure_parameters
-            ))
-        .isNotEmpty;
+    return orbp.getBoilerplateDeclarations(orbp.detectBoilerplateMembers(unit), null).isNotEmpty;
```

Comment:
Did this stuff get removed, or just moved?

Reply:
Removed; this was a case of this part of the description:
> Make builder's parseDeclarations/getBoilerplateDeclarations's ErrorCollector arg optional so we don't have to construct noops ErrorCollectors to pass in, and clean up existing noop ErrorCollectors in analyzer plugin

This ErrorCollector was a noop, and now we can just pass in `null` instead of setting one up.